content/en/docs/how-to/registry-authentication.md

```diff
@@ -7,74 +7,54 @@ weight: 1
 
 You can use `notation` to authenticate to an OCI-compliant registry. This is useful for pushing and pulling signed artifacts, as well as inspecting artifacts in registries that do not offer public access.
 
-{{% alert title="Important" color="important" %}}
-Currently, `notation` relies on [Docker Credential Store](https://docs.docker.com/engine/reference/commandline/login/#credentials-store) for authentication. Notation requires additional configuration for Docker credential helper if you are using Notation in Linux.
-{{% /alert %}}
-
-## Use `notation login` to authenticate to an OCI-compliant registry
+When you authenticate to an OCI-compliant registry, `notation`, by default, uses a [credentials store](#configure-a-credentials-store) to store your credentials in a secure manner. It is possible to store your credentials in [environment variables](#configure-environment-variables-to-authenticate-to-an-oci-compliant-registry) or as [base64 encoded values in a configuration file](#configure-configjson-to-authenticate-to-an-oci-compliant-registry), but these approaches are not secure and are not recommended for production environments. **It is strongly recommended that you use a credentials store to authenticate to an OCI-compliant registry.**
 
-To authenticate to an OCI-compliant registry, use the `notation login` command with your registry, username, and password. For example:
-
-```console
-notation login -u <username> -p <password> <registry>
-```
-
-{{% alert title="Note" color="primary" %}}
-If `notation login` is failing, you may need to configure [Docker Credential Store](https://docs.docker.com/engine/reference/commandline/login/#credentials-store) as detailed in the [Configure Docker Credential Store for Linux](#configure-docker-credential-store-for-linux) section.
-
-In development and testing environments, you can use [environment variables](#configure-environment-variables-to-authenticate-to-an-oci-compliant-registry) to authenticate to an OCI-compliant registry.
-{{% /alert %}}
+## Configure a credentials store
 
-## Configure Docker Credential Store for Linux
+A credentials store is a secure container that stores your credentials, such as usernames and passwords. Operating systems often include a default credentials store, such as *osxkeychain* on macOS or *wincred* on Windows. You can also use a third-party credentials store, such as [pass](https://www.passwordstore.org/).
```

Comment:
nit: Suggest not using the word "container" to describe a credential store. It will confuse customers because Notation is to sign container images. You can use other words to describe a credential store.

Reply:
I debated using *container* in this instance, but i couldn't immediately think of a better way to phrase it.

I'm open to suggestions.

Reply:
fixed